knowledge_base/database_with_catalog/databricks.yml

```diff
@@ -0,0 +1,28 @@
+bundle:
+  name: database-catalog-example
+
+# workspace:
+#   host: https://myworkspace.cloud.databricks.com
+
+resources:
+  database_instances:
+    my_instance:
+      name: my-instance
+      capacity: CU_1
+  database_catalogs:
+    my_catalog:
+      database_instance_name: ${resources.database_instances.my_instance.name}
+      name: example_catalog
+      database_name: my_database
+      create_database_if_not_exists: true
```

Comment:
What does this field do? Can this be omitted if the database is already being created as part of the deployment?

Reply:
this creates a postgres database named my_database if it does not exist on the instance yet. so it can be omitted in certain cases, but not because this deployment creates a database instance.